### PR TITLE
Add logical turtle post for tommytester

### PR DESCRIPTION
**Pull request by tommytester**
I'm not sure I did everything right. Pay attention on this code review please!

### CODE REVIEW

File: _posts/tommytester/2020-09-09-tommytester-logical-turtle [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+author:Tommy
+layout: post
+Title: Tommy's logical turtle
+---
+
+Here's my logical turtle:
+https://trinket.io/python/715e82971d
```
Comment (cchinchia) [inline _posts/tommytester/2020-09-09-tommytester-logical-turtle]: This should be an embed code

File: _posts/tommytester/2020-09-09-tommytester-logical-turtle [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+author:Tommy
+layout: post
+Title: Tommy's logical turtle
```
Comment (cchinchia) [inline _posts/tommytester/2020-09-09-tommytester-logical-turtle]: lowercase the title, and the content should be in quotation

File: _posts/tommytester/2020-09-09-tommytester-logical-turtle [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+author:Tommy
```
Comment (cchinchia) [inline _posts/tommytester/2020-09-09-tommytester-logical-turtle]: add space between semicolon and tommy

Comment (cchinchia) [reply _posts/tommytester/2020-09-09-tommytester-logical-turtle]: Oh, and "tommy" should be the username

**Review by cchinchia — CHANGES_REQUESTED** [review]
Looks great and I left a few comments for you :)

**Comment by cchinchia** [comment]
one more thing: add .md

**Comment by tommytester** [comment]
@cchinchia Thanks for the comments- ready for a second review

**Review by cchinchia — APPROVED** [review]
Wow 👯

**Pull request merged**